Makefile

```diff
@@ -192,3 +192,11 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+.PHONY: start-minikube
+start-minikube:
```

Comment:
@djzager We gotta ensure these get reconciled when it is time to re-scaffold the operator

Reply:
Speaking in ignorance, is that something to do with a future changes required when picking up newer ansible-operator versions?

Reply:
When we pickup newer ansible-operator , we tend to re-scaffold in that newer version which causes Makefile among many other files to be overwritten , so that's why is a good idea at least to make a mental note of the new targets for CI.